File: scripts/delete-lockfile.py
import os
import sys

lockfile_path = '/pnpm-lock.yaml'

try:
    if os.path.exists(lockfile_path):
        os.remove(lockfile_path)
        print(f"Successfully deleted {lockfile_path}")
    else:
        print(f"{lockfile_path} not found")
except Exception as e:
    print(f"Error deleting {lockfile_path}: {e}")
    sys.exit(1)

File: scripts/delete-lockfile.mjs
#!/usr/bin/env node
import fs from 'fs';
import path from 'path';

const lockfilePath = path.join(process.cwd(), 'pnpm-lock.yaml');

try {
  if (fs.existsSync(lockfilePath)) {
    fs.unlinkSync(lockfilePath);
    console.log('✓ Successfully deleted pnpm-lock.yaml');
  } else {
    console.log('ℹ pnpm-lock.yaml not found - already deleted');
  }
} catch (error) {
  console.error('✗ Error deleting pnpm-lock.yaml:', error.message);
  process.exit(1);
}
